ci/docs-build-verification.sh

```diff
@@ -13,19 +16,19 @@ function isDocsFile() {
     if echo ${FILE_PATH} | grep ".*.md" ; then
         return 0
     fi
-    
+
     return 1
 }
 
 function cleanUp() {
     # Ensure we are not in the temp dir before cleaning it
-    cd ${CURRENT_DIR} 
+    cd ${CURRENT_DIR}
```

Comment:
I only removed the trailing whitespace, it wasn't me!

Sure, `shellcheck`'ing. Please would you try the change, however, since I am not running it. Either that, or I'll revert all changes from this file in this branch as out of scope.